### DISCUSSION TITLE
Myth of Empires

### DISCUSSION THREAD

**Discussion by Blacky244-dev** [BattlefieldDuck/WindowsGSM#184] [General]
Hello since I don't know if it belongs here I just wanted to ask how it looks with the support of Myth of Empires that the game is now back in Steam and your server manager could take some of the work off I just wanted to ask if you would add the game your manager is just clearer and easier to understand especially the autoupdate would save me some work on the server

**Comment by dat-data** [comment]
I'm new to WindowsGSM so pardon my ignorance but it looks like this is already being achieved via a plugin.  I recommend heading over to Sarpendon's repo and checking out the install instructions.  Feel free to give it a star if it works for you 👍

https://github.com/Sarpendon/WindowsGSM.MythofEmpires
